modules/OFStateManager/module/src/gentable_handlers.c

```diff
@@ -185,45 +185,49 @@ indigo_core_gentable_unregister(indigo_core_gentable_t *gentable)
 
 /* OpenFlow message handlers */
 
+/**
+ * gentable add/modify resume functon
+ * Drivers will call this resume function if the async operation reaches
+ * the final status
+ * If it is not an async operation, the indigo will call it.
+ * (The existing non-async operation deson't need to modify the codes.)
```

Comment:
"existing non-async operations do not need any modification and do not call resume functions" or some similar english